database-commons/src/main/java/io/cdap/plugin/db/sink/AbstractDBSink.java

```diff
@@ -170,18 +225,22 @@ public void prepareRun(BatchSinkContext context) {
       configAccessor.setTransactionIsolationLevel(dbSinkConfig.getTransactionIsolationLevel());
     }
 
-    // Get Hadoop configuration object
-    Configuration configuration = configAccessor.getConfiguration();
-
     // Configure batch size if specified in pipeline arguments.
     if (context.getArguments().has(ETLDBOutputFormat.COMMIT_BATCH_SIZE)) {
       configuration.set(ETLDBOutputFormat.COMMIT_BATCH_SIZE,
                         context.getArguments().get(ETLDBOutputFormat.COMMIT_BATCH_SIZE));
     }
 
+    addOutputContext(context);
```

Comment:
make sure use of 'this' not required here. test it once that it is calling child class